Setting these flags is wrong and breaks the build:
cc1plus: error: unrecognized command line option "-stdlib=libstdc++"

--- x2goclient.pro	2023-06-29 04:09:48.000000000 +0800
+++ x2goclient.pro	2023-12-31 07:27:02.000000000 +0800
@@ -242,11 +242,6 @@
   message("building $$TARGET with ldap and cups")
   LIBS += -framework LDAP -lcups -lcrypto -lssl -lz
 
-  !isEmpty(OSX_STDLIB) {
-    QMAKE_CXXFLAGS += -stdlib=$${OSX_STDLIB}
-    QMAKE_LFLAGS += -stdlib=$${OSX_STDLIB}
-  }
-
   !isEmpty(MACPORTS_INCLUDE_PATH) {
     INCLUDEPATH += $${MACPORTS_INCLUDE_PATH}
   }
